ui/design-system-core/src/components/Text/index.tsx

```diff
@@ -47,11 +47,11 @@ const VARIANT_TO_CSS_CLASSES_MAPPER: Record<Variant, string> = {
   body1: 'text-[1rem] leading-[1.5rem]',
   body2: 'text-[0.875rem] leading-[1.375rem]',
   label: 'text-[1rem] leading-[1.5rem] font-medium',
-  footnotes1: 'text-[0.75rem] leading-[1.125rem] font-bold tracking-[.075em]',
+  footnotes1: 'text-[0.75rem] leading-[1.125rem] font-bold tracking-[.075em] font-normal',
```

Comment:
I had removed `font-normal` here, it is the default by tailwind, also it gives room for specifying any other font weight through customStyle

also check, `font-bold` had already been specified before `font-normal`, so you may remove both classes

Reply:
I like to make it specific and not rely on defaults in general :) In addition, we have `apply` function to override any custom style definition.

Reply:
okay, but font-bold was also defined

you may have to remove it instead

`...font-bold tracking-[.075em] font-normal`